crates/network-scanner/Cargo.toml

```diff
@@ -8,12 +8,20 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+dashmap = "5.5.3"
+derive_builder = "0.13.0"
```

Comment:
issue: We are already using typed-builder in the rest of the codebase, I would prefer we stick to this

Reply:
updated